src/SearchEngine/MASA.BuildingBlocks.SearchEngine.AutoComplete/Response/ResponseBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace MASA.BuildingBlocks.SearchEngine.AutoComplete.Response;
+public class ResponseBase
+{
+    public bool IsValid { get; }
+
+    public string Message { get; }
+
+    public ResponseBase(bool isValid, string message)
+    {
+        IsValid = isValid;
+
+        if (isValid)
+        {
+            message = "success";
+        }
+
+        Message = message;
```

Comment:
Message = isValid ? "success" : message;

Reply:
Another point, success like magic.

Reply:
Here is to control the success of the response message is the same, but it seems a bit redundant now